### PR TITLE
Remove placeholder/template text

### PR DESCRIPTION
**Pull request by TomLav**
The text this PR removes appears to be a placeholder/template content added by a web content manager (Rise ?) software.

see https://www.reddit.com/r/instructionaldesign/comments/17zpyvh/when_we_show_up_to_the_present_moment_with_all_of/?rdt=57853

### CODE REVIEW

File: docs/Area2_Capacity_Sharing/Open-Science-101/modules/Module_3/Lesson_2/readme.md [hunk 1]
```diff
@@ -401,4 +399,4 @@ What information is commonly found in a citation file?
 - Version
 - URL
 - Creation date
-- All of the above
```
Comment (TomLav) [inline docs/Area2_Capacity_Sharing/Open-Science-101/modules/Module_3/Lesson_2/readme.md:404]: This last line of the M3/L2 `readme.md` should *not* be changed by this commit.

**Comment by bressler95tops** [comment]
Thanks @TomLav part of this makes sense with removing the rise placeholder text.  One thing I have to ask about is what the thought process might have been with removing the "All of The Above" answer from the last knowledge check question?

**Comment by bressler95tops** [comment]
Or actually never mind, I just checked and saw that you had amended that line back in!

**Comment by TomLav** [comment]
@bressler95tops. I do not understand why Github marks the last line "All of the above" as modified. I would have preferred my PR to only show the text deletion, it would be cleaner.

Could it be because there is no newline at the end of this last line, and that this mixes Github's diff?

**Comment by katblanchette** [comment]
@bressler95tops - Please move forward with the fix. Thanks!

**Comment by bressler95tops** [comment]
@katblanchette I have reviewed this and it looks good to me. Feel free to review the 'files changed' tab above if you want to give things a double check as well, or have the MOOC developers do so, so they can get a more precise view of what changed. Let me know when you have had a chance to work this into the MOOC as well so we can officially check this off.

**Comment by bressler95tops** [comment]
Updating status as this has been completed on the MOOC now.

**Pull request merged**